src/tests/excalidrawPackage.test.tsx

```diff
@@ -130,4 +130,66 @@ describe("<Excalidraw/>", () => {
       expect(textInput?.nodeName).toBe("SPAN");
     });
   });
+
+  describe("Test UIOptions prop", () => {
+    it('should not hide any UI element when the UIOptions prop is "undefined"', async () => {
```

Comment:
let's add a describe block for `Test canvasActions` nested inside this describe as in future we will be adding more props.

Reply:
Got it!